### PR TITLE
[refactor] #4230: Remove `Domain::logo`

### LINKED ISSUE #4230: Remove `Domain::logo`
Domain logo should be stored as a metadata value

### PR DESCRIPTION
**Pull request by dima74**
## Description

Remove Domain::logo. Users who need it could store logo in domain metadata

### Linked issue

Closes #4230

### Checklist

- [x] I've read CONTRIBUTING.md
- [x] I've used the standard signed-off commit format (or will squash just before merging)
- [ ] All applicable CI checks pass (or I promised to make them pass later)
- [ ] (optional) I've written unit tests for the code changes
- [ ] I replied to all comments after code review, marking all implemented changes with thumbs up

### CODE REVIEW

File: data_model/src/domain.rs [hunk 1]
```diff
@@ -16,7 +16,6 @@ pub use self::model::*;
 use crate::{
     account::{Account, AccountsMap},
     asset::{AssetDefinition, AssetDefinitionsMap, AssetTotalQuantityMap},
-    ipfs::IpfsPath,
```
Comment (Erigara) [inline data_model/src/domain.rs:19]: If `IpfsPath` isn't no longer used we can remove it.
Going to create separate ticket.

Comment (dima74) [reply data_model/src/domain.rs:19]: It is used by `AssetDefinition::logo`. Should `AssetDefinition::logo` also be removed?

Comment (Erigara) [reply data_model/src/domain.rs:19]: yes, the same logic

Comment (dima74) [reply data_model/src/domain.rs:19]: Ok, will remove `AssetDefinition::logo` as part of #4417

**Review by s8sato — DISMISSED** [review]

**Comment by Asem-Abdelhady** [comment]
i think the problem with the the conflicts is that you have to keep your version for executor.wasm

**Pull request closed without merging**